packages/datetime/src/dateRangePicker.tsx

```diff
@@ -35,6 +35,14 @@ import { TimePicker } from "./timePicker";
 export interface IDateRangeShortcut {
     label: string;
     dateRange: DateRange;
+
+    /**
+     * By default, clicking a shortcut does not change the time of the date range picker, but instead
+     * takes the date components of the `dateRange` and combines it with the currently selected time.
+     * Setting `shouldChangeTime` to `true` will override this behavior and allow shortcuts to change
+     * the selected time as well.
+     */
```

Comment:
nit: Might be self-evident, but may be worth calling out that the default is `false`:

```tsx
/**
 * ...
 * the selected time as well.
 * @default false
 */
```

Reply:
done